pkgs/applications/misc/mkgmap/default.nix

```diff
@@ -14,11 +14,11 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "mkgmap";
-  version = "4835";
+  version = "4837";
 
   src = fetchurl {
     url = "https://www.mkgmap.org.uk/download/mkgmap-r${version}-src.tar.gz";
```

Comment:
It looks like we should switch to `fetchsvn`, since upstream provides tarball only for latest version, e.g. `https://www.mkgmap.org.uk/download/mkgmap-r4835-src.tar.gz` is no longer available.

Reply:
Could you do this?

Reply:
Yes, I'll do with next update.